tests/test_encode_decode.py

```diff
@@ -95,8 +96,80 @@ def test_b85_encode_decode_round_trip(self, payload, pad):
 
 
 class TestBinASCII(unittest.TestCase):
-    # TODO: https://docs.python.org/3/library/binascii.html
-    pass
+    @given(payload=st.binary(), backtick=st.booleans())
+    def test_b2a_uu_a2b_uu_round_trip(self, payload, backtick):
+        x = binascii.b2a_uu(payload, backtick=backtick)
```

Comment:
[The `backtick` option is new in Python 3.7](https://docs.python.org/3/library/binascii.html#binascii.b2a_uu), so we'll want a condition like `if sys.version_info[:2] >= (3, 7):` and only pass it in the first branch.

Reply:
Sure 👍